src/components/calcite-pick-list/shared-list-tests.ts

```diff
@@ -17,7 +17,7 @@ export const tests = {
 
         await item1.click();
         await item2.click();
-        expect(toggleSpy).toHaveReceivedEventTimes(2);
+        expect(toggleSpy).toHaveReceivedEventTimes(listType === "pick" ? 3 : 4);
```

Comment:
Boulder: This is an error. The events are getting duplicated with the code like this.

There may be a simpler way - If we change the attribute name that the state reflects.
```selected={this.isSelected}```
to
```isSelected={this.isSelected}```

Or we can remove it entirely. Now that we're using props and not attributes, the functionality doesn't depend on the existence of this attribute.

Reply:
I'll research and see why its being called more for value list.

Reply:
Found out that the items were not properly being matched. Now the event call count is the same because the value is what is being matched.

Reply:
All edits to tests reverted. Fixed issues.